app/app/src/main/kotlin/com/hedvig/app/feature/profile/ui/myinfo/MyInfoDestination.kt

```diff
@@ -0,0 +1,177 @@
+      navigateUp = navigateUp,
+      modifier = Modifier.clearFocusOnTap(),
+    ) {
+      Spacer(Modifier.height(16.dp))
+      val errorText = uiState.member?.phoneNumber?.errorMessageRes?.let { stringResource(id = it) }
+      HedvigTextField(
+        value = uiState.member?.phoneNumber?.input ?: "",
+        onValueChange = { newInput ->
+          if (newInput.indices.all { newInput[it].isWhitespace().not() }) {
+            phoneNumberChanged(newInput)
+          }
+        },
+        label = { Text(stringResource(R.string.PHONE_NUMBER_ROW_TITLE)) },
+        errorText = errorText,
+        keyboardOptions = KeyboardOptions(
+          keyboardType = KeyboardType.Phone,
+          imeAction = ImeAction.Next,
+        ),
+        modifier = Modifier
+          .fillMaxWidth()
+          .padding(horizontal = 16.dp),
+      )
+      AnimatedVisibility(visible = errorText != null) {
+        Spacer(Modifier.height(4.dp))
+      }
+      Spacer(Modifier.height(4.dp))
+      HedvigTextField(
+        value = uiState.member?.email?.input ?: "",
+        onValueChange = { newInput ->
+          if (newInput.indices.all { newInput[it].isWhitespace().not() }) {
+            emailChanged(newInput)
+          }
+        },
+        label = { Text(stringResource(R.string.PROFILE_MY_INFO_EMAIL_LABEL)) },
+        errorText = uiState.member?.email?.errorMessageRes?.let { stringResource(id = it) },
+        keyboardOptions = KeyboardOptions(
+          keyboardType = KeyboardType.Email,
+          imeAction = ImeAction.Done,
+        ),
+        keyboardActions = KeyboardActions(
+          onDone = {
+            updateEmailAndPhoneNumber()
+            localSoftwareKeyboardController?.hide()
+          },
+        ),
+        modifier = Modifier
+          .fillMaxWidth()
+          .padding(horizontal = 16.dp),
+      )
+      Spacer(Modifier.height(16.dp))
+      AnimatedVisibility(
+        visible = uiState.canSubmit || uiState.isSubmitting,
+        enter = fadeIn(),
+        exit = fadeOut(),
+      ) {
+        HedvigContainedButton(
+          text = stringResource(R.string.general_save_button),
+          enabled = uiState.canSubmit,
+          onClick = {
+            updateEmailAndPhoneNumber()
+            localSoftwareKeyboardController?.hide()
+          },
+          isLoading = uiState.isSubmitting,
+          modifier = Modifier.padding(horizontal = 16.dp),
+        )
+        Spacer(Modifier.height(16.dp))
+      }
+
+      AnimatedVisibility(
+        visible = uiState.errorMessage != null,
+        enter = fadeIn(),
+        exit = fadeOut(),
+      ) {
+        Column {
+          uiState.errorMessage?.let {
+            WarningTextWithIcon(
+              modifier = Modifier
+                .padding(horizontal = 16.dp)
+                .fillMaxWidth()
+                .wrapContentWidth(),
+              text = it,
+            )
+          }
+          Spacer(Modifier.height(16.dp))
+        }
+      }
```

Comment:
This will show error messages from the back-end. Perhaps it makes more sense to use the general error screen, here: https://www.figma.com/file/WlwFCAWywXNcXVgPfYVmWu/Final-Design-App?node-id=1654%3A29136&mode=dev